src/java.desktop/share/classes/javax/swing/JPasswordField.java

```diff
@@ -280,6 +280,30 @@ public String getText(int offs, int len) throws BadLocationException {
         return super.getText(offs, len);
     }
 
+    @Override
+    @BeanProperty(bound = false, description = "the text of this component")
+    public void setText(String t) {
+        // overwrite the old data first
```

Comment:
If we dont have javadoc for a public method, wouldn't build like make-docs fail?

Reply:
"make docs" works fine.